internal/pkg/agent/cmd/inspect.go

```diff
@@ -35,8 +35,8 @@ import (
 func newInspectCommandWithArgs(s []string, streams *cli.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "inspect",
-		Short: "Shows configuration of the agent",
-		Long: `Shows current configuration of the agent.
+		Short: "Show current configuration of the Elastic Agent",
+		Long: `This command will show the current configuration of the Elastic Agent.
```

Comment:
```suggestion
		Long: `This command shows the current configuration of the Elastic Agent.
```
We try to use present tense whenever possible.